tests/data_files/Makefile

```diff
@@ -276,6 +276,9 @@ all_final += server5-ss-forgeca.crt
 server5-othername.crt: server5.key
 	$(OPENSSL) req -x509 -new -subj "/C=UK/O=Mbed TLS/CN=Mbed TLS othername SAN" -set_serial 77 -config $(test_ca_config_file) -extensions othername_san -days 3650 -sha256 -key $< -out $@
 
```

Comment:
A bug fix in the library needs a non-regression test (present) and a [changelog entry](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md) (missing). Please add a changelog entry.

Reply:
Hello @gilles-peskine-arm, thank you for the reminder. I added a bugfix changelog entry as requested.